dir.traversal.targets

```diff
@@ -77,6 +77,9 @@
     <Error Condition="'$(MSBuildLastTaskResult)'=='false'" />
   </Target>
 
+  <!-- Dumpling.targets needs to install dumpling.py before running tests. -->
+  <Import Project="$(ToolsDir)Dumpling.targets" Condition="'$(EnableDumpling)' == 'true' OR '$(EnableCloudTest)' == 'true'" />
```

Comment:
Also, is this file imported by upload-tests.proj? from a very quick scan it didn't seem like it is and as far as I know, EnableCloudTest is only set to true for that project.